game/engine.py

```
@@ -30,6 +30,8 @@
 class ChessGame:
     """Manage a single chess game: state, validation,
       and engine communication."""
+    
+    DIFFICULTY_DEPTH_MAP = {'easy': 1, 'medium': 2, 'hard': 3}
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)